projects/aca-shared/src/lib/components/info-drawer/info-drawer.component.ts

```diff
@@ -37,6 +37,7 @@ import { InfoDrawerModule } from '@alfresco/adf-core';
 import { TranslateModule } from '@ngx-translate/core';
 import { A11yModule } from '@angular/cdk/a11y';
 import { ToolbarComponent } from '../toolbar/toolbar.component';
+import { NodeActionsService } from '../../../../../aca-content/src/lib/services/node-actions.service';
```

Comment:
incorrect import again

Reply:
Updated and removed the circular dependency.